package.json

```diff
@@ -48,7 +48,7 @@
     "tslib": "^2.4.1"
   },
   "devDependencies": {
-    "@fpjs-incubator/broyster": "^0.1.3",
+    "@fpjs-incubator/broyster": "^0.1.4",
```

Comment:
Please run `yarn install` to update `yarn.lock` and push the new version of `yarn.lock`.